include/vrb/Matrix.h

```diff
@@ -91,6 +91,26 @@ class Matrix {
     return result;
   }
 
+  static Matrix Rotation(const Vector& aDirection, const Vector& aUp = vrb::Vector(0.0f, 1.0f, 0.0f)) {
+    Matrix result;
+    const vrb::Vector dir = aDirection.Normalize();
```

Comment:
You need to then use the dir var bellow when calculating the result as well I assume?

Reply:
Yes, forgot about it. PR updated